README.md

```diff
@@ -1038,6 +1038,7 @@ This list is the result of Pull Requests, reviews, ideas and work done by 1100+
 ## BaaS
 
   * [Activepieces](https://www.activepieces.com) - Build automation flows to connect several apps together in the backend of your app. For example, when an event fires in your app, send a Slack message or add a Google Sheet row. Free up to 5,000 tasks per month.
+  * [Appwrite](https://appwrite.io/) - Flexible and scalable backend for web and mobile apps. Appwrite includes Authentication, Storage, Realtime Database and Cloud Functions.
```

Comment:
I dont see any details here about pricing or free tier, can you please link to a pricing page or one detailing what's free?